src/frontend/pages/landing-page/LandingPage.jsx

```diff
@@ -0,0 +1,138 @@
+
+
+export default function LandingPage (){
+    return <div>
+       <NavgationTop/>
+    <main className="wrapper flex-column">
+        <div className="full-wrapper flex-column">
+           <div className="background-img-container">
+               <img src={landing_image} alt="backgroung-image" className="responsive-img" />
+                <div className="centered-content">
+                    <p className="m-text">Sometimes you will never know the <strong>true value</strong> of a moment unit
+                    it becomes <strong>a memory.</strong></p>
+                    <p className="m-text">So we at <span className="text-italic font-weight-4">Shutter Island</span> aim at catering all your photography needs in a single marketplace at a reliable and discounted prices.</p> 
+                    <Link to="/categories"><button className="btn primary-btn mg-t-m">Shop Products</button></Link>
+                </div>
+           </div>
+        </div>
+
+        <div className="overlay-card-container flex-column">
+                <div className="flex-sb mg-t-s mg-l-m">
+                    <div className="overlay-card pointer border-radius-sm flex-se">
+                        <img src={canon_eos_6d} alt="EOS 6D" className="responsive-image"/>
+                        <div className="overlay-text">
+                            <p className="m-text">Camera</p>
+                        </div>
+                        
+                    </div>
+                    <div className="flex-center-column mg-r-m">
+                        <h2>Camera</h2>
+                        <p>"Life is like a camera. Focus on what's important. Capture the good times. And if things don't work out, just take another shot."</p>
+                        <a href="../products/products-listing-1.html"><button className="btn primary-btn mg-t-xs">Shop Cameras</button></a>
+                    </div>
+                </div>
+                
+                <div className="line-divider"></div>
+
+                <div className="flex-sb mg-t-s mg-r-m">
+                    <div className="flex-center-column mg-l-m">
+                        <h2>Lens</h2>
+                        <p>“Before you point fingers at someone, clean them well. You better remove that log on your lens before you can see the speck on someone's own afar!”</p>
+                        <a href="../products/products-listing-1.html"><button className="btn primary-btn mg-t-xs">Shop Lens</button></a>
+                    </div>
+                    <div className="overlay-card pointer border-radius-sm flex-se">
+                        <img src={nikkor_200mm} alt="NIKKOR_200MM" className="responsive-image"/>
+                        <div className="overlay-text">
+                            <p className="m-text">Lens</p>
+                        </div>
```

Comment:
Your LandingPage looks quite big, there is repeated use of card. It would be good if you extract it into a separate component and map them using some data structure.